NMS-13364: Table formatting issues in new docs (#3531)
OpenNMS/opennms: docs/modules/operation/pages/service-assurance/monitors/FtpMonitor.adoc

> = FtpMonitor

Reviewer: I think the Hint section at the very bottom probably can be removed from this document, otherwise the sentence structure needs cleanup.

> The FtpMonitor validates FTP connection dial-up processes.

Reviewer: ```suggestion
The FtpMonitor validates an FTP connection's login processes.
```

The monitor can test the FTP server on multiple ports with specific login data.

The service using the FtpMonitor is up if the FTP server responds with return codes between 200 and 299.
For special cases, the service is also marked as up for codes 425 and 530.

This monitor implements <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-placeholder-substitution-parameters, placeholder substitution in parameter values>>.

== Monitor facts

[options="autowidth"]
|===
| Class Name     | `org.opennms.netmgt.poller.monitors.FtpMonitor`
| Remote Enabled | `true`
|===

== Configuration and use

.Optional monitor-specific parameters for the FtpMonitor
[options="header"]
[cols="1,3,2"]

|===
| Parameter              | Description                                                                                 |Default
| retry                | Number of attempts to get a valid FTP response/response-text.                                | 0
| port                 | A list of TCP ports to which connection shall be tried.                                     | 20, 21 
| password

> *{}*             | Specifies the password to use. Use this parameter with the userid parameter to perform

Reviewer: ```suggestion
*{}*             | Specifies the password to use.
                   Use this parameter with the userid parameter to perform authentication.                         | empty string
```

>                            authentication.                         | empty string 

Reviewer: ```suggestion
```

| userid  

*{}*

>           | Specifies the user ID to use. Use this parameter with the password parameter to perform

Reviewer: ```suggestion
          | Specifies the user ID to use.
            Use this parameter with the password parameter to perform authentication.   |n/a
```

>                            authentication.                            |n/a 

Reviewer: ```suggestion
```
